src/ConductorSharp.Toolkit/Service/ScaffoldingService.cs

```diff
@@ -0,0 +1,286 @@
+            {
+                note = "The autogenerated name has been changed because it is invalid C# member name";
+            }
+            //name = char.ToUpperInvariant(name[0]) + name[1..];
+
+            string workflowDescription = workflowDefinition.Description;
+            string[] labels = null;
+
+            try
+            {
+                var descriptionObject = JObject.Parse(workflowDescription);
+                workflowDescription = descriptionObject.SelectToken("description")?.Value<string>();
+                var labelsObject = descriptionObject.SelectToken("labels");
+
+                if (labelsObject is JArray labelsArray)
+                {
+                    labels = labelsArray.ToObject<string[]>();
+                }
+            }
+            catch (Exception) { }
+
+            var inputLinesBuilder = new StringBuilder();
+            try
+            {
+                var inputParameters = JObject.Parse(workflowDefinition.InputParametersJSON[0]);
+
+                foreach (var param in inputParameters)
+                {
+                    var value = param.Value.SelectToken("value")?.ToString(Newtonsoft.Json.Formatting.None);
+                    var type = param.Value.SelectToken("type")?.Value<string>();
+                    var description = param.Value.SelectToken("description")?.Value<string>();
+
+                    if (type == null)
+                        type = "string";
+
+                    inputLinesBuilder.AppendLine($"        /// <originalName>");
+                    inputLinesBuilder.AppendLine($"        /// {param.Key}");
+                    inputLinesBuilder.AppendLine($"        /// </originalName>");
+
+                    if (!string.IsNullOrEmpty(description))
+                    {
+                        inputLinesBuilder.AppendLine($"        /// <summary>");
+                        inputLinesBuilder.AppendLine($"        /// {description}");
+                        inputLinesBuilder.AppendLine($"        /// </summary>");
+                    }
+
+                    string propertyName;
+
+                    switch (type)
+                    {
+                        case "string":
+                        case "integer":
+                            if (!string.IsNullOrEmpty(value))
+                            {
+                                inputLinesBuilder.AppendLine($"        /// <remark>");
+                                inputLinesBuilder.AppendLine($"        /// Example: {value}");
+                                inputLinesBuilder.AppendLine($"        /// </remark>");
+                            }
+                            inputLinesBuilder.AppendLine($"        [JsonProperty(\"{param.Key}\")]");
+                            LangUtils.MakeValidMemberName(SnakeCaseUtil.ToPascalCase(param.Key), "A", out propertyName);
+                            inputLinesBuilder.AppendLine($"        public dynamic {propertyName} {{ get; set; }}");

```

Comment:
We can extract this to a method, something like 
`AppendClassProperty(string type, string name)`

Reply:
Will do it, but do not want to make it extension method since that means it has to go into separate static class and that means we separate implementation from main class (it is better for related functionality to exist in same class). Hence I will add method in ScaffoldingService.